sdk/iothub/Microsoft.Azure.Management.IotHub/tests/Helpers/IotHubTestUtilities.cs

```diff
@@ -13,20 +13,20 @@ namespace IotHub.Tests.Helpers
 {
     public class IotHubTestUtilities
     {
-        public static string DefaultLocation = "WestUS2";
-        public static string DefaultFailoverLocation = "WestCentralUS";
-        public static string DefaultIotHubName = "DotNetHub";
-        public static string DefaultUpdateIotHubName = "UpdateDotNetHub";
-        public static string DefaultResourceGroupName = "DotNetHubRG";
-        public static string DefaultUpdateResourceGroupName = "UpdateDotNetHubRG";
-        public static string EventsEndpointName = "events";
-        public static string DefaultCertificateIotHubName = "DotNetCertificateHub";
-        public static string DefaultCertificateResourceGroupName = "DotNetCertificateHubRG";
-        public static string DefaultIotHubCertificateName = "DotNetCertificateHubCertificate";
-        public static string DefaultIotHubCertificateSubject = "Azure IoT Root CA";
-        public static string DefaultIotHubCertificateThumbprint = "9F0983E8F2DB2DB3582997FEF331247D872DEE32";
-        public static string DefaultIotHubCertificateType = "Microsoft.Devices/IotHubs/Certificates";
-        public static string DefaultIotHubCertificateContent = "MIIBvjCCAWOgAwIBAgIQG6PoBFT6GLJGNKn/EaxltTAKBggqhkjOPQQDAjAcMRow"
+        public const string DefaultLocation = "WestUS2";
+        public const string DefaultFailoverLocation = "WestCentralUS";
+        public const string DefaultIotHubName = "DotNetHub";
+        public const string DefaultUpdateIotHubName = "UpdateDotNetHub";
+        public const string DefaultResourceGroupName = "DotNetHubRG";
+        public const string DefaultUpdateResourceGroupName = "UpdatedDotNetHubRG";
+        public const string EventsEndpointName = "events";
+        public const string DefaultCertificateIotHubName = "DotNetCertificateHub";
+        public const string DefaultCertificateResourceGroupName = "DotNetCertificateHubRG";
+        public const string DefaultIotHubCertificateName = "DotNetCertificateHubCertificate";
+        public const string DefaultIotHubCertificateSubject = "Azure IoT Root CA";
+        public const string DefaultIotHubCertificateThumbprint = "9F0983E8F2DB2DB3582997FEF331247D872DEE32";
+        public const string DefaultIotHubCertificateType = "Microsoft.Devices/IotHubs/Certificates";
+        public const string DefaultIotHubCertificateContent = "MIIBvjCCAWOgAwIBAgIQG6PoBFT6GLJGNKn/EaxltTAKBggqhkjOPQQDAjAcMRow"
                                                              + "GAYDVQQDDBFBenVyZSBJb1QgUm9vdCBDQTAeFw0xNzExMDMyMDUyNDZaFw0xNzEy"
```

Comment:
just curious, is it okay storing the cert ?

Reply:
It is used temporarily in a hub that is created, and shortly after removed. It should be fine.